tools/building.py

```diff
@@ -697,6 +697,8 @@ def acorn_optimizer(filename, passes, extra_info=None, return_output=False):
   # will be carried over to a later Closure run.
   if Settings.USE_CLOSURE_COMPILER:
     cmd += ['--closureFriendly']
+  if Settings.VERBOSE:
+    cmd += ['verbose']
```

Comment:
Why does `--closureFriendly` look like an option but `verbose` does not?  Wouldn't it cleaner to have all options start with `--`?

Reply:
Good question, I'm not sure. It's possible that was copied over from the old JS optimizer so there is a mix of notations. Also some of these (like `last`) used to be passes and became flags. (Anyhow, it is an internal API so it's not urgent to fix I think.)